packages/jellyfish-api-core/src/category/wallet.ts

```diff
@@ -62,6 +62,16 @@ export class Wallet {
     return await this.client.call('getbalance', ['*', minimumConfirmation, includeWatchOnly], 'bignumber')
   }
 
+  /**
+   * Identical to getBalance to get untrusted pending balance
+   *
+   * @param {boolean} withTokens
```

Comment:
@param {boolean} [withTokens=false]
This is an optional value
See https://jsdoc.app/tags-param.html#optional-parameters-and-default-values